### DISCUSSION TITLE
Does it support the new protocol?

### DISCUSSION THREAD

**Discussion by caoyang728** [swar/Swar-Chia-Plot-Manager#1250] [Q&A]
Does it support the new protocol? 
What Settings should I change compared to before, such as which should be used for farmer_public_key and pool_public_key?
Looking forward to your reply. Thank you

**Comment by Chilli71** [comment]
I assume you mean pools.
The current version supports pools.
Create the NTF in the gui.
Delete  the pool_public_key value in the manager config if you reuse an existing config.
Add the pool contract address copied from the ntf in the config (at pool_contract_address)
Thats all.
Happy farming.
